Goal: fix the path of the utmp and wtmp files

Fixes: 501353

Status wrt upstream: Will be fixed in 4.1.3

--- a/man/ja/login.1
+++ b/man/ja/login.1
@@ -112,9 +112,9 @@
 .PP
 システム設定の違いによって上記ファイルの置き場所は変わる。
 .SH ファイル
-\fI/var/log/utmp\fR		\- 現在のログインセッションのリスト
+\fI/var/run/utmp\fR		\- 現在のログインセッションのリスト
 .br
-\fI/var/run/wtmp\fR		\- 過去のログインセッションのリスト
+\fI/var/log/wtmp\fR		\- 過去のログインセッションのリスト
 .br
 \fI/etc/passwd\fR		\- ユーザアカウント情報
 .br
